Library/Formula/anttweakbar.rb

```diff
@@ -1,5 +1,5 @@
 class Anttweakbar < Formula
-  desc "C/C++ library for adding GUIs to OpenGL apps"
+  desc "C/C++ library for adding GUIs 往 OpenGL apps"
```

Comment:
I'm seriously lost though. I performed `git reset --hard origin/master`, which added the newline characters.... So not sure what's happening here, honestly.